tests/suites/0_stateless/02_0008_function_crashme.sql

```diff
@@ -0,0 +1,2 @@
+SELECT 1;
+SELECT crashme(sleep(1));
```

Comment:
maybe we should also add `SELECT 1` after `SELECT crashme(sleep(1))` ?

Reply:
Good point, but the mysql client seems not work in the fuse-test.

Reply:
For the query `SELECT * FROM (SELECT crashme(sleep(1)))`, the clients get 3 times ERROR 2013:
```
mysql> SELECT * FROM (SELECT crashme(sleep(1)));
ERROR 2013 (HY000): Lost connection to MySQL server during query
No connection. Trying to reconnect...
Connection id:    8
Current database: *** NONE ***

ERROR 2013 (HY000): Lost connection to MySQL server during query
No connection. Trying to reconnect...
Connection id:    8
Current database: *** NONE ***

ERROR 2013 (HY000): Lost connection to MySQL server during query

```

Reply:
Create an issue here: https://github.com/datafuselabs/datafuse/issues/919